src/BookReader.js

```diff
@@ -377,7 +377,7 @@ BookReader.prototype.initParams = function() {
   // Check for Search plugin
   if (this.options.enableSearch) {
     // Go to first result only if no default or URL page
-    this.goToFirstResult = !params.pageFound
+    this.options.goToFirstResult = !params.pageFound
```

Comment:
`this.goToFirstResult` was incorrect because we don't have any property like this. This change will carry the `this.options.goToFirstResult` value to determine if we need to jump on first search result in search.plugin.js file